docs/NSR.md

```diff
@@ -17,21 +17,21 @@ a.	Segment Certificate Systems into networks based on their functional or logica
 
 b.	Apply equivalent security controls to all systems co-located in the same network with a Certificate System;
 
-c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks;
+c.	Maintain Root CA Systems in a High Security Zone and in an offline state or air-gapped from all other networks. Offline Root CA systems shall be a non-networked, stand-alone system that is powered down when not in use; 
 
 d.	Maintain and protect Issuing Systems, Certificate Management Systems, and Security Support Systems in at least a Secure Zone;
 
 e.	Implement and configure Security Support Systems that protect systems and communications between systems inside Secure Zones and High Security Zones, and communications with non-Certificate Systems outside those zones (including those with organizational business units that do not provide PKI-related services) and those on public networks;
 
 f.	Configure each network boundary control (firewall, switch, router, gateway, or other network control device or system) with rules that support only the services, protocols, ports, and communications that the CA has identified as necessary to its operations;
 
-g.	Configure Issuing Systems, Certificate Management Systems, Security Support Systems, and Front-End / Internal-Support Systems by removing or disabling all accounts, applications, services, protocols, and ports that are not used in the CA’s or Delegated Third Party’s operations and allowing only those that are approved by the CA or Delegated Third Party;
+g.	Configure all systems used during a certificate life-cycle event by removing or disabling all accounts, applications, services, protocols, and ports that are not used in the CA’s or Delegated Third Party’s operations and allowing only those that are approved by the CA or Delegated Third Party;
 
-h.	Review configurations of Issuing Systems, Certificate Management Systems, Security Support Systems, and Front-End / Internal-Support Systems on at least a weekly basis to determine whether any changes violated the CA’s security policies;
+h.	Review configurations of Issuing Systems, Certificate Management Systems, Security Support Systems, and Front-End / Internal-Support Systems on at least a weekly basis to determine whether any changes violated the CA’s security policies. For offline Root CA systems, review configurations every 30 days or when the system is powered-on;
 
 i.	Grant administration access to Certificate Systems only to persons acting in Trusted Roles and require their accountability for the Certificate System’s security;
 
-j.	Implement Multi-Factor Authentication to each component of the Certificate System that supports Multi-Factor Authentication;
+j.	Implement Multi-Factor Authentication to each component of the Certificate System that supports Multi-Factor Authentication. For offline Root CA systems, Multi-Factor Authentication should be implemented and based on compensating access controls;
```

Comment:
As long as they keep: "that supports Multi-Factor Authentication;" I'm ok with the original requirement.